### PR TITLE
12565 - SBC-Common-Components - Logout not clearing SiteMinder cookies

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/12565

*Description of changes:*
Change login to log out of SiteMinder as well. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: vue/sbc-common-components/src/services/keycloak.services.ts [hunk 1]
```diff
@@ -116,6 +116,7 @@ class KeyCloakService {
               resolve()
             }
             redirectUrl = redirectUrl || `${window.location.origin}${process.env.VUE_APP_PATH}`
+            redirectUrl = `https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`
```
Comment (seeker25) [inline vue/sbc-common-components/src/services/keycloak.services.ts:119]: I was hoping we could make this configurable somewhere that way we could change it disable it?  Any ideas?

File: vue/sbc-common-components/src/services/keycloak.services.ts [hunk 2]
```diff
@@ -116,6 +116,9 @@ class KeyCloakService {
               resolve()
             }
             redirectUrl = redirectUrl || `${window.location.origin}${process.env.VUE_APP_PATH}`
+            if (logoutSiteMinder) {
+              redirectUrl = `https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`
```
Comment (shabeeb-aot) [inline vue/sbc-common-components/src/services/keycloak.services.ts:120]: What will be the URL for prod siteminder?

**Comment by shabeeb-aot** [comment]
might want to npm install and update package-lock.json also

File: vue/sbc-common-components/src/services/keycloak.services.ts (hunk 2 above)
Comment (seeker25) [reply vue/sbc-common-components/src/services/keycloak.services.ts:120]: That is the prod URL. I don't think the TEST one works correctly.

**Comment by seeker25** [comment]
ok, let me npm install and update package-lock.json.

**Comment by seeker25** [comment]
We discussed we need this URL passed in from session storage. If it doesn't exist, don't provide the extra logout step. 
Also there is a TEST version and a PROD version:

TEST example:
`https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`

PROD example:
`https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`

**Review by shabeeb-aot — APPROVED** [review]

**Comment by shabeeb-aot** [comment]
We need add SITEMINDER_LOGOUT_URL to vault

**Pull request merged**